src/components/input/input.scss

```diff
@@ -104,9 +131,16 @@ $md-input-underline-disabled-background-image: linear-gradient(to right,
       &.md-accent {
         color: $md-input-underline-color-accent;
       }
+      &.md-warn {
+        color: $md-input-underline-color-warn;
+      }
     }
   }
 
+  input:-webkit-autofill + .md-input-placeholder {
```

Comment:
Add comment like 
```scss
// Pseudo-class for Chrome and Safari auto-fill to move the placeholder to
// the floating position. This is necessary because these browsers do not actually
// fire any events when a form auto-fill occurs.
```